boot-qemu.py

```diff
@@ -764,13 +764,19 @@ def launch_qemu(cfg):
                 utils.die("Port 1234 is already in use, is QEMU running?")
 
             utils.green("Starting QEMU with GDB connection on port 1234...")
-            with subprocess.Popen(qemu_cmd + ["-s", "-S"]) as qemu_process:
+            qemu_cmd += ["-s", "-S"]
```

Comment:
Is this just going to add `-s -S` to the QEMU invocation every time it gets restarted? I think this probably wants to be outside of the loop.

Reply:
good catch!